src/components/Admins.vue

```diff
@@ -24,7 +24,7 @@
 		<span :class="{ hidden: !isLoading }" class="icon icon-loading" />
 		<div v-for="admin in admins" :key="admin.id" class="admin-avatar-container">
 			<NcAvatar :user="admin.internal ? admin.id : null"
-				:display-name="admin.displayname"
+				:title="admin.displayname"
```

Comment:
This should be upstreamed to the vue lib, so it automatically is done for all other usages of the component?